pennylane/math/utils.py

```diff
@@ -195,6 +195,14 @@ def get_interface(*values):
 
     * Vanilla NumPy arrays and SciPy sparse matrices can be used alongside other tensor objects;
       they will always be treated as non-differentiable constants.
+
+    .. warning: ``get_interface`` defaults to ``"numpy"`` whenever Python built-in objects are passed.
+        I.e. a list or tuple of ``torch`` tensors will be identified as ``"numpy"``:
+
+        >>> get_interface([torch.tensor([1]), torch.tensor([1])])
+        "numpy"
+
+        The correct usage in that case is to unpack the arguments ``get_interface(*[torch.tensor([1]), torch.tensor([1])])``.

```

Comment:
```suggestion
    .. warning:: 
        ``get_interface`` defaults to ``"numpy"`` whenever Python built-in objects are passed.
        I.e. a list or tuple of ``torch`` tensors will be identified as ``"numpy"``:

        >>> get_interface([torch.tensor([1]), torch.tensor([1])])
        "numpy"

        The correct usage in that case is to unpack the arguments ``get_interface(*[torch.tensor([1]), torch.tensor([1])])``.

```

Reply:
Thanks! Hadnt checked the rendered version yet 👍